packages/design-system/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+import { useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+
+import styles from './Breadcrumbs.module.scss';
+import Link from '../Link';
+import Dropdown from '../Dropdown/Dropdown';
+import { ButtonTertiary } from '../Button';
+import { StackHorizontal } from '../Stack';
+import Divider from '../Divider';
+
+type BreadcrumbsLink = {
+	label: string;
+	href: string;
+	target?: string;
+};
+
+type BreadcrumbsRouterLink = {
+	label: string;
+	target?: string;
+	as: ReactElement;
+};
+
+type BreadcrumbsItem = (BreadcrumbsRouterLink | BreadcrumbsLink)[];
+
+type BreadCrumbsProps = Omit<HTMLAttributes<HTMLElement>, 'className' | 'style'> & {
+	items: BreadcrumbsItem;
+};
+
+const maxBreadcrumbsItemLength = 4;
+
+const Breadcrumbs = forwardRef(({ items, ...rest }: BreadCrumbsProps, ref: Ref<HTMLElement>) => {
+	const { t } = useTranslation();
+	const buildEntries = () => {
+		if (items.length > maxBreadcrumbsItemLength) {
+			const origin = items[0];
+			const suffix = items.slice(-2);
+			const collapsed = items.slice(1, items.length - 2);
+
+			return (
+				<>
+					<li className={styles.entry}>
+						<StackHorizontal gap="S" align="center" wrap="nowrap">
+							{'href' in origin ? (
+								<Link href={origin.href} target={origin.target} withEllipsis>
+									{origin.label}
+								</Link>
+							) : (
+								<Link as={origin.as} target={origin.target} withEllipsis>
+									{origin.label}
+								</Link>
+							)}
+
+							<span aria-hidden className={styles.divider}>
+								<Divider orientation="vertical" />
+							</span>
+						</StackHorizontal>
+					</li>
+
+					<li className={classnames(styles.entry, styles.collapsed)}>
+						<StackHorizontal gap="S" align="center" wrap="nowrap">
+							<Dropdown
+								aria-label={t('COLLAPSED_LINKS_MENU', 'Collapsed links')}
```

Comment:
The aria-label doesn't seem to be used, we end up with a button without aria-label, leaving `...` as only text

Reply:
The label is not on the button, it's on the menu. The label describes the menu.

Reply:
Pushed something that I think is best